scio-bigquery/src/test/scala/com/spotify/scio/bigquery/types/Schemas.scala

```diff
@@ -71,4 +71,8 @@ object Schemas {
   case class RepeatedNested(required: List[Required],
                             optional: List[Optional],
                             repeated: List[Repeated])
+
+  case class User(@description("user name") name: String, @description("user age") age: Int)
+  case class Account(@description("account user") user: User,
+                     @description("in USD") balance: Double)
```

Comment:
I moved these to the companion object because the `class SchemaProviderTest extends FlatSpec with Matchers` wont be serializable because `org.scalatest.FlatSpec` isn't serializable. Not sure if this is the right way to do this. What are your thoughts.

I was getting the following exception if I had in in the class and not in the companion object: 
```
[info]   Cause: java.io.NotSerializableException: org.scalatest.Assertions$AssertionsHelper
[info]  - field (class "org.scalatest.FlatSpec", name: "assertionsHelper", type: "class org.scalatest.Assertions$AssertionsHelper")
[info]  - object (class "com.spotify.scio.bigquery.types.TypeProviderTest", TypeProviderTest)
[info]  - field (class "com.spotify.scio.bigquery.types.TypeProviderTest$S1", name: "$outer", type: "class com.spotify.scio.bigquery.types.TypeProviderTest")
[info]  - root object (class "com.spotify.scio.bigquery.types.TypeProviderTest$S1", S1(1))
[info]   at java.io.ObjectOutputStream.writeObject0(ObjectOutputStream.java:1182)
```
I enabled `-Dsun.io.serialization.extendedDebugInfo=true` to trace this.

Reply:
LGTM